packages/client/src/workspace/dock.ts

```diff
@@ -37,7 +37,7 @@ export const registerDock = () => {
     ],
   }
 
-  Dock.register(dockProvider)
+  await Dock.register(dockProvider)
```

Comment:
surprisingly crucial bit .. without this, it just won't start up
OpenFin must have fixed a bug in their async behaviour, as this code has been like this for ages .. 🙄